utils/utils.go

```diff
@@ -50,6 +50,7 @@ func ConvertAPIStringToDate(date string) time.Time {
 func SendErrorBack(w http.ResponseWriter, err error, info string) {
 	log.Println(info + ":", err)
 	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusBadRequest);
```

Comment:
I guess its just for now, if theres any error the server would return always a bad request ?

Reply:
For now that's what I have but I can update the utils function to take in a http status code like the SendResponseBack function so we can send back something other than bad request.